### ISSUE TITLE
[Question]: Missing arguments class_weight and sample_weight in model.fit()

### ISSUE THREAD

**Issue by FrancescoRusticali** [SciSharp/TensorFlow.NET#1177]
### Description

Hi all,
I noticed that the method model.fit() is missing arguments class_weight and sample_weight (coming from the corresponding Python function), that are used for weighting the loss function, usually to deal with imbalanced datasets.
Is there a reason for that?
Any workaround to apply similar weights (the class weights in particular)?

Thank you very much!

**Comment by Wanglongzhi2001** [comment]
Hello, the implementation of class_weight and sample_weight has completed in #1187 #1189 
Please let me know if there still exist some problems.
